stormpath/express-stormpath
# Add multi-tenancy support

Adds support for multi-tenancy.

#### How to verify

Use the example application below:

```javascript
var express = require('express');
var stormpath = require('express-stormpath');

var app = express();

app.use(stormpath.init(app, {
  debug: 'info',
  web: {
    me: {
      enabled: true
    },
    multiTenancy: {
      enabled: true
    }
  }
}));

app.on('error', function (err) {
  console.error(err);
});

app.listen(3000, function () {
  console.log('Listening on http://localhost:3000/!');
});
```

After starting the application, navigate to `/register` and create a new account for an existing organization. Once the account has been created, navigate to `/login` and login with your new account in the same organization. Once logged in, navigate to `/me` and verify that the organization that you specified is present in the JSON result.

Fixes #292
